enigma-core/app/src/networking/ipc_listener.rs

```diff
@@ -353,9 +351,16 @@ pub(self) mod handling {
                 // Save the ExeCode into the DB.
                 let key = DeltaKey::new(contract_address, Stype::ByteCode);
                 db.create(&key, &v.output)?;
-                Ok(v.into_deploy_response(&bytecode))
+                let wasm_result = v.into_deploy_response(&bytecode);
```

Comment:
`wasm_result` is not a good name since it contains an IpcResponse

Reply:
Done